src/pp-annotator/api/DebugStorage.ts

```diff
@@ -24,36 +24,38 @@ export default class DebugStorage implements IStorage, IModule {
     }
   }
 
-  create(annotation: IAnnotation): IAnnotation {
+  create(annotation: IAnnotationAPIModel): IAnnotationAPIModel {
     if (!annotation.id) {
       annotation.id = generateId();
     }
+    annotation.create_date = new Date(Date.now()).toISOString();
```

Comment:
I think `new Date()` by default has the value of `Date.now()`. The latter just returns a `number` instead of `Date object`

Reply:
That's right (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date), I'll change that.